sp800-63b/sec10_usability.md

```diff
@@ -6,7 +6,7 @@
 
 _This section is informative._
 
-[ISO/IEC 9241-11](#ISO9241) defines usability as the "extent to which a product can be used by specified users to achieve specified goals with effectiveness, efficiency and satisfaction in a specified context of use." This definition focuses on users, their goals, and the context of use as key elements necessary for achieving effectiveness, efficiency, and satisfaction. A holistic approach that accounts for these key elements is necessary to achieve usability.
+[ISO/IEC 9241](#ISO9241) defines usability as the "extent to which a product can be used by specified users to achieve specified goals with effectiveness, efficiency and satisfaction in a specified context of use." This definition focuses on users, their goals, and the context of use as key elements necessary for achieving effectiveness, efficiency, and satisfaction. A holistic approach that accounts for these key elements is necessary to achieve usability.
```

Comment:
Apologies if we're out of sync here -- what's in square brackets should read [ISO/IEC 9241-11] as it was before. This is correct because it's part 11, not version 11. You can leave the hashtag the way it was.